src/components/Calculator.jsx

```diff
@@ -1,33 +1,66 @@
 /* eslint-disable linebreak-style */
-/* eslint-disable react/no-access-state-in-setstate */
-/* eslint-disable react/destructuring-assignment */
-/* eslint-disable linebreak-style */
-/* eslint-disable indent */
+/* eslint-disable react/no-array-index-key */
+/* eslint-disable arrow-body-style */

```

Comment:
- [OPTIONAL] As requested by the previous code reviewer, please make sure you don't disable `eslint`, except it is absolutely necessary as this is to help you implement best practices. This is like modifying the `linter` which is prohibited.  Please kindly fix that.